src/components/Layout.tsx

```diff
@@ -0,0 +1,35 @@
+import classNames from 'classnames';
+import { NavLink, Outlet } from 'react-router-dom';
+
+const getClass = ({ isActive }: { isActive: boolean }) =>
+  classNames('navbar-item', { 'has-background-grey-lighter': isActive });
+
+export const Layout = () => {
+  return (
+    <>
+      <nav
+        data-cy="nav"
+        className="navbar is-fixed-top has-shadow"
+        role="navigation"
+        aria-label="main navigation"
+      >
+        <div className="container">
+          <div className="navbar-brand">
+            <NavLink className={getClass} to="/">
```

Comment:
You can move all routes in Enum, and then use them in all places in your application.